polartools/load_data.py

```diff
@@ -208,3 +216,256 @@ def is_Bluesky_specfile(source, folder=""):
+    """
+    Extracts metadata of a list of scans.
+
+    Parameters
+    ----------
+    scans : iterable
+        Scan numbers or uids.
+    db : databroker database
+        Searcheable database
+    scan_to : int, optional
+        Final scan number to process. Note that this is only meaningful if
+        an integer is passed to `scans`.
+    meta_keys : iterable
+        List with metadata keys to read.
+    query : dictionary, optional
+        Search parameters.
+
+    Returns
+    -------
+    meta : dictionary
+        Metadata organized by scan number or uid (whatever is given in
+        `scans`).
+    """
+    db_range = db_query(db, query=query) if query else db
+    output = OrderedDict()
+    for scan in scan_numbers:
+        try:
+            start = db_range[scan].metadata["start"]
+            stop = db_range[scan].metadata.get("stop", None)
+
+            output[scan] = OrderedDict()
+            for key in meta_keys:
+                output[scan][key] = [start.get(key, None)]
+                if stop is None:
+                    output[scan][key] += [None]
+                else:
+                    output[scan][key] += [stop.get(key, None)]
+                output[scan][key] = _flatten_list(output[scan][key])
+
+        except KeyError:
+            warn(f"The scan number {scan} was not found.")
+
+    return output
+
+
+def _flatten_list(inp):
+    """Only handles lists to second level"""
+    output = []
+    for item in inp:
+        if isinstance(item, list):
+            output += [i for i in item]
+        else:
+            output.append(item)
+    return output
+
+
+def lookup_position(db, scan, search_string="", query=None):
+    """
+    Lookup positioner values in past scans.
+
+
+    Parameters
+    ----------
+
+    db : databroker database
+        Searcheable database
+    scan : integer
+        Scan numbers or uids.
+    search_string : string
+        Full or part of positioner name.
+    query: dict
+        Search parameters.
+
+
+    Returns
+    -------
+    output: list
+
+    """
+
+    db_range = db_query(db, query=query) if query else db
+
+    status = db_range[scan].metadata["stop"]
+    if status and status["exit_status"] == "success":
+        baseline = load_databroker(scan, db, "baseline")
+        date1 = baseline["time"][1].strftime("%m/%d/%y %H:%M:%S")
+        date2 = baseline["time"][2].strftime("%m/%d/%y %H:%M:%S")
+        print("=".center(100, "="))
+        print(f"{'Positioner':>50}{date1:>25}{date2:>25}")
+        print("-".center(100, "-"))
+        for key in baseline.keys():
+            if search_string in key:
+                if isinstance(baseline[key][1], list):
+                    print(f"{key:>50}{baseline[key][1]}{baseline[key][2]}")
+                else:
+                    print(
+                        f"{key:>50}{baseline[key][1]:>25}{baseline[key][2]:>25}"
+                    )
+
+    elif not status:
+        raise ValueError("Scan not existing!")
```

Comment:
I think this is not true, the baseline may exist (with 1 point), but `status = None`

Reply:
Changed wording in if none statement to 'possibly' as well. It is not possible to take care of all possibilities directly. If this error message appears direct access of the baseline may still reveal some information.